packages/storage/src/types/upload.ts

```diff
@@ -53,6 +53,17 @@ export type IpfsUploaderOptions = {
   apiKey?: string;
 };
 
+/**
+ * @public
+ */
+export type IpfsDownloaderOptions = {
+  /**
+   * Optional apiKey to associate with the IpfsDownloader.
+   * You can get an API key here: https://thirdweb.com/settings
```

Comment:
The url should be `https://thirdweb.com/dashboard/`, in case we want to change it later